Chaincase.UI/Pages/NewPasswordPage.razor

```diff
@@ -14,24 +14,28 @@
     <h1>Set your password</h1>
     <IonItem>
         <ion-label floating>Password</ion-label>
-        <IonInput type='@(showPassword ? "text" : "password")' @bind-Value="@Password" placeholder="e.g. margaretthatcheris110%SEXY" />
-        @if (showPassword)
+        <IonInput type='password' @bind-Value="@Password" placeholder="e.g. margaretthatcheris110%SEXY" />
+        @if (!string.IsNullOrEmpty(error))
```

Comment:
:( what's wrong with hide show? 

Empty password should not be an error case. Warning -> (proceed || go back) at most obtrusive.

Reply:
There is already  a hide/show in ionic for password fields.

Reply:
Separate PR with these stuff now